WebUI/war/views/PercUserView.js

```diff
@@ -92,13 +92,19 @@
             $("#perc-users-edit-user-button").off("click").on("click",function(){
                 controller.editSelectedUser();
                 addingNewUser = false;
+            });
+			$("#perc-users-edit-user-button").off("keydown").on("keydown",function(){
+               document.activeElement.click();
```

Comment:
@ramdev1980 Question on these click handlers.  Shouldn't it be narrowed to the Enter key?  Otherwise if you press the tab key that would also trigger the button click right?